packages/game-app/src/_shared/index.ts

```diff
@@ -18,6 +22,13 @@ export function bootstrap() {
     messagingSenderId: CONFIG.REACT_APP_FIREBASE_CONFIG_MESSAGING_SENDER_ID,
     appId: CONFIG.REACT_APP_FIREBASE_CONFIG_APP_ID,
   });
+  if (config.REACT_APP_FIREBASE_USE_EMULATORS === 'true') {
+    firebase.firestore().settings({ experimentalForceLongPolling: true });
+    firebase.auth().useEmulator('http://localhost:9099/');
```

Comment:
Why is this line using the full URL while the others don't?

Reply:
firebase exposes inconsistent api for the useEmulator. You can see the doc

Reply:
lol